templates/embeds/card.html.twig

```diff
@@ -12,8 +12,9 @@
 {% set _title               = block('box_title') is defined ? block('box_title') : null %}
 {% set _tools               = block('box_tools') is defined ? block('box_tools') : null %}
 {% set _header              = block('box_header') is defined ? block('box_header') : null %}
+{% set _margin_bottom       = margin_bottom is defined ? margin_bottom : 'mb-3' %}
```

Comment:
Something tickle me with the variable name VS what's in.

With the current code, `margin_bottom` sounds like true/false. 
But it expect class name.

Without seeing the code, I was expecting a number to be given.
Like `{margin_bottom : 3}`.

What do you think ?
```twig
{% set _margin_bottom       = margin_bottom is defined ? 'mb-' ~ margin_bottom : 'mb-3' %}
```

Reply:
I am fine with your solution: if there is a `mb-0` class!? 
I just made it this way, so it can be removed entirely by providing an empty string (which is not very elegant tbh).

Reply:
There is a mb-0 class.